testsuite/tests/lexing/Makefile

```diff
@@ -0,0 +1,18 @@
+#**************************************************************************
```

Comment:
Please don't include copyright headers in the test suite, I'm trying to get rid of them.